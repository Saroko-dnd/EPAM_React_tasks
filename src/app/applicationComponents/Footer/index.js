import React from 'react';

import './Footer.scss';

const Footer = () => (
    <footer className="card site-footer d-flex align-items-center flex-column">
        <p>Developer: Saroko Igor Viktorovich</p>
        <p>Powered by News API</p>
    </footer>
);

export default Footer;
