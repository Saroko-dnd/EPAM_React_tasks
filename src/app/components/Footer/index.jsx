import React from 'react';

import './scss/Footer.scss';

const Footer = () => (
    <footer className="card site-footer d-flex align-items-center flex-column">
        <p className="d-none d-sm-block">Developer: Saroko Igor Viktorovich</p>
        <p className="d-block d-sm-none">Developer: Saroko I.V.</p>
        <p>Powered by News API</p>
    </footer>
);

export default Footer;
