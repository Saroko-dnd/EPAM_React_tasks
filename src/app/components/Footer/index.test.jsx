import Footer from '.';

test('render a Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});
