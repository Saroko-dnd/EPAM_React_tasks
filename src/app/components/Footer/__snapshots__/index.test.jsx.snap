// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Footer should render Footer correctly 1`] = `
<footer
  className="card site-footer d-flex align-items-center flex-column"
>
  <p
    className="d-none d-sm-block"
  >
    Developer: Saroko Igor Viktorovich
  </p>
  <p
    className="d-block d-sm-none"
  >
    Developer: Saroko I.V.
  </p>
  <p>
    Powered by News API
  </p>
</footer>
`;
